cluster/src/dunit/scala/io/snappydata/cluster/QueryRoutingDUnitTest.scala

```diff
@@ -337,12 +354,12 @@ class QueryRoutingDUnitTest(val s: String) extends ClusterManagerTestBase(s) {
     }
     assert(foundTable)
 
-    val rSet2 = dbmd.getTables(null, JDBCAppendableRelation.INTERNAL_SCHEMA_NAME, null,
+    val rSet2 = dbmd.getTables(null, INTERNAL_SCHEMA_NAME + "APP", null,
```

Comment:
Ok, looks like you are now adding table schema name to the internal column table schema name. Is that correct?